apps/api/src/app/subscribers/subscribers.controller.ts

```diff
@@ -301,8 +301,8 @@ export class SubscribersController {
       subscriberId: subscriberId,
       environmentId: user.environmentId,
       templateId: templateId,
-      channel: body.channel,
-      enabled: body.enabled,
+      ...(typeof body.enabled === 'boolean' && { enabled: body.enabled }),
+      ...(body.channel && { channel: body.channel }),
```

Comment:
Protection against wrong payloads.

Reply:
Is used in other controller endpoints so for now I'd rather not touch it, just in case.